supabase/config.toml

```diff
@@ -0,0 +1,334 @@
+# For detailed configuration reference documentation, visit:
+# https://supabase.com/docs/guides/local-development/cli/config
+# A string used to distinguish different Supabase projects on the same host. Defaults to the
+# working directory name when running `supabase init`.
+project_id = "supa-db-migration"
```

Comment:
This should be removed, no? Let it default to the working directory? Should we always call supabase init when creating a new worktree?